modules/issue_tracker/ajax/EditIssue.php

```diff
@@ -195,7 +195,7 @@ function validateInput($values)
         $validCenter = $db->pselectOne(
             "
             SELECT
-                CenterID = :center_id
+                CenterID = :center_id as CenterID
```

Comment:
Please change CenterID -> RegistrationCenterID

Reply:
@xlecours Good catch!! This is fixed now.